packages/stream/common/utils.ts

```diff
@@ -27,3 +29,20 @@ export const getNumberFromBN = (value: BN, decimals: number): number =>
   value.gt(new BN(2 ** 53 - 1))
     ? value.div(new BN(10 ** decimals)).toNumber()
     : value.toNumber() / 10 ** decimals;
+
+export async function handleError<T>(
```

Comment:
This is more or less contract error wrapper, so I am thinking should name of the func have contract in it?